airflow/www/static/css/main.css

```diff
@@ -269,3 +269,10 @@ div.square {
 .sd { color: #BA2121; font-style: italic } /* Literal.String.Doc */
 .s2 { color: #BA2121 } /* Literal.String.Double */
 .s1 { color: #BA2121 } /* Literal.String.Single */
+
+.highlight--demo-mode .s,
```

Comment:
Previously this was on `pre span.s` - do we need these three selectors, would just `.highlight--demo-mode` be enough?

Reply:
(What are .s1 and .s2? Is there a .s3 for instance?)

Reply:
The previous implementation did not blur any string.  Probably it was adapted to the old version of Pygments. Look at the style code for pygments a few lines above. I note that styles s1 and s2 are at bottom, so they were probably added later. The current version blurs the text in each case.

Reply:
Oh yes. Right in front of me!

Reply:
I will mention that I am working on a patch that deletes this CSS code copied from library pygments.
https://github.com/mik-laj/incubator-airflow/pull/8/commits/90a4be93536e6cbe236856b10d0fc9c5788d371d
However, this is still in the task queue